sp/src/game/server/hl2/npc_combine.cpp

```diff
@@ -1114,6 +1114,40 @@ bool CNPC_Combine::CanOrderSurrender()
 
 	return m_iCanOrderSurrender == TRS_TRUE;
 }
+
+ConVar npc_combine_gib( "npc_combine_gib", "0" );
+
+//-----------------------------------------------------------------------------
+// Purpose: 
+// Input  : &info - 
+// Output : Returns true on success, false on failure.
+//-----------------------------------------------------------------------------
+bool CNPC_Combine::ShouldGib( const CTakeDamageInfo &info )
```

Comment:
I'm wondering if we should add a condition for "not temporal"? I wouldn't want gibbing to interfere with temporal death effects in the nemesis fight.

Reply:
I was unaware this was a possibility.